miso-web/src/main/webapp/scripts/sequencer_partition_container_ajax.js

```diff
@@ -282,7 +282,7 @@ Container.ui = {
       {
         'updateElement': 'containerdiv',
         'doOnSuccess': function(json) {
-            jQuery('#containerdiv').find('input[type="radio"]').first().click();
+          jQuery('#containerdiv').find('input.selectMe').click();
```

Comment:
You should be able to put `checked="checked"` on the radio button you want and not bother with this jQuery.